src/js/messaging/containers/Folder.jsx

```diff
@@ -334,33 +335,48 @@ export class Folder extends React.Component {
           this.props.fetchFolder(lastRequest.id, lastRequest.query);
         };
 
-        return (
-          <p>
-            Could not retrieve the folder.&nbsp;
-            <a onClick={reloadFolder}>Click here to try again.</a>
-          </p>
+        componentContent = (
+          <div className="columns">
```

Comment:
I think it's from Foundation because the design team is using their grid. I usually use that if it's not worth defining our own container/div classes just for the left + right padding